mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java

```diff
@@ -398,11 +398,14 @@ public boolean canGoBack() {
      * @return true if back key was consumed, false otherwise
      */
     public boolean goBack() {
-        if (mTemporaryPage instanceof FullScreenWebviewFragment) {
-            WebView webView = ((FullScreenWebviewFragment) mTemporaryPage).mWebview;
+        WebView webView = mTemporaryPage instanceof FullScreenWebviewFragment
+                ? ((FullScreenWebviewFragment) mTemporaryPage).mWebview : null;
+        if (webView != null) {
             if (webView.canGoBack()) {
                 webView.goBack();
                 return true;
+            } else {
+                webView.loadUrl("about:blank");
```

Comment:
What's the point of this? The WebView is going to be killed off when the fragment is removed anyway.

Reply:
If I go back only once I see the sitemap, but the fragment isn't killed. There are still log messages from HABPanel.